File: app/src/main/java/com/neome/java/api/core/user/msg/MsgGrantBearerToken.java
// neome.ai API - do not change
//

package com.neome.java.api.core.user.msg;

import com.neome.java.api.meta.base.Types.TabId;
import com.neome.java.api.nucleus.base.Types.AppVersion;
import com.neome.java.api.nucleus.base.msg.Msg;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class MsgGrantBearerToken extends Msg
{
  public AppVersion appVersion;

  @Nullable
  public Boolean sendCaller;

  public TabId tabId;
}

File: app/src/main/java/com/neome/api/core/user/msg/MsgGrantBearerToken.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.core.user.msg

import com.neome.api.nucleus.base.Types.AppVersion
import com.neome.api.nucleus.base.msg.Msg
import com.neome.api.meta.base.Types.TabId

interface MsgGrantBearerToken : Msg
{
  val appVersion: AppVersion
  val sendCaller: Boolean?
  val tabId: TabId
}